src/ray/raylet/raylet_client.h

```diff
@@ -53,6 +53,17 @@ typedef
 
 namespace ray {
 
+/// Interface for pinning objects. Abstract for testing.
+class PinObjectsInterface {
+ public:
+  /// Request to a raylet to pin a plasma object. The callback will be sent via gRPC.
+  virtual ray::Status PinObjectIDs(
```

Comment:
Hmm why not use gRPC for both?

Reply:
Er I'm not sure, I was just reusing what was already here.

Reply:
Nvm, this does go over grpc. Comment may be misleading, this method executes a grpc request exchange.